### ISSUE TITLE
[html/css] Skiplinks

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#6559]
### 📝 Description
The Skip Links component already exists in v10 documentation, but its current styling is minimal and was implemented prior to having finalized design specifications.

### ♿ Accessibility
Skip links are essential for users who navigate via keyboard, screen readers, or switch controls.
Accessibility requirements include:
- Skip links must be focusable via the Tab key and visible on focus.
- Must point to a valid, unique target (e.g., `<main id="main-content">`).
- Must be the first focusable element on the page.
- Should support multiple skip links if there are multiple navigation targets (e.g., main content, navigation, footer).

Note that since this is an HTML/CSS component, those accessibility requirements should be implemented in the preview of the component as well as stated in the documentation.

### 🧪 Testing

_Tests have already been implemented for this component._

### 🔗 References
[GOV.UK](https://design-system.service.gov.uk/components/skip-link/)
[SBB.CH](https://digital.sbb.ch/en/design-system/lyne/components/skiplink-list/)

### 📃 Tasks

- [ ] Implement styles according to design
- [ ] Update storybook documentation if information is missing regarding usage
- [ ] Design review

**Comment by gfellerph** [comment]
Re-triage due to design change
